### PR TITLE
Change data preparation from class to function

### PR DESCRIPTION
**Pull request by pplantinga**
A bit nicer interface for calling the preparation. Instead of:

```python
preparer = TIMIT_Preparer(...)
preparer()
```
Just do:
```python
prepare_timit(...)
```
- [x] timit
- [x] librispeech

### CODE REVIEW

File: recipes/LibriSpeech/librispeech_prepare.py [hunk 1]
```diff
@@ -42,298 +50,317 @@ class LibriSpeechPreparer:
...
-    >>> LibriSpeechPreparer(data_folder,splits,save_folder)
+    >>> data_folder = 'datasets/LibriSpeech'
+    >>> splits = ['train-clean-100', 'dev-clean', 'test-clean']
+    >>> save_folder = 'librispeech_prepared'
+    >>> prepare_librispeech(data_folder, splits, save_folder)
     """
+    data_folder = data_folder
+    splits = splits
+    save_folder = save_folder
+    select_n_sentences = select_n_sentences
```
Comment (csukuangfj) [inline recipes/LibriSpeech/librispeech_prepare.py:61]: This line does not make sense.

File: recipes/LibriSpeech/librispeech_prepare.py [hunk 2]
```diff
@@ -42,298 +50,317 @@ class LibriSpeechPreparer:
...
+        )
 
-            split = self.splits[split_index]
+        text_dict = text_to_dict(text_lst)
 
-            wav_lst = get_all_files(
-                self.data_folder + "/" + split, match_and=[".flac"]
-            )
+        if select_n_sentences is not None:
+            select_n_sentences = select_n_sentences[split_index]
```
Comment (csukuangfj) [inline recipes/LibriSpeech/librispeech_prepare.py:104]: The documentation of this function says `select_n_sentences` is an `int`,
how can it support the index operator `[]`?

**Comment by csukuangfj** [comment]
The above two issues still exist in the latest master.

File: recipes/LibriSpeech/librispeech_prepare.py (hunk 1 above)
Comment (pplantinga) [reply recipes/LibriSpeech/librispeech_prepare.py:61]: You're right they should be removed.

File: recipes/LibriSpeech/librispeech_prepare.py (hunk 2 above)
Comment (pplantinga) [reply recipes/LibriSpeech/librispeech_prepare.py:104]: You're right, the `select_n_sentences` variable is not used and should be removed. Feel free to open a PR for that!

**Pull request merged**